### DISCUSSION TITLE
Welcome to showinfilemanager Discussions!

### DISCUSSION THREAD

**Discussion by damonlynch** [damonlynch/showinfilemanager#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas about how best to develop this Python module.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amykyta3** [comment]
Just wanted to say - Thanks for the tool! Exactly what I was looking for and it works wonderfully

**Comment by damonlynch** [comment]
Thanks for your kind words!
